pkg/registry/bundle.go

```diff
@@ -262,23 +286,42 @@ func (b *Bundle) cache() error {
 		}
 	}
 
-	if b.crds == nil {
-		b.crds = []*v1beta1.CustomResourceDefinition{}
+	if b.v1beta1crds == nil {
+		b.v1beta1crds = []*apiextensionsv1beta1.CustomResourceDefinition{}
 	}
+	if b.v1crds == nil {
+		b.v1crds = []*apiextensionsv1.CustomResourceDefinition{}
+	}
+
 	for _, o := range b.Objects {
 		if o.GetObjectKind().GroupVersionKind().Kind == "CustomResourceDefinition" {
```

Comment:
Maybe should put this CRD string to be a constant as well.

Reply:
I wonder if we need this at all - the `Bundle` struct has the uninitialized version of the CRD array attached to it when it gets created.